### ISSUE TITLE
downloading from XNAT crashes FSLeyes 1.2.0

### ISSUE THREAD

**Issue by dmd** [pauldmccarthy/fsleyes#86]
In `fsleyes/FSLeyes version 1.2.0`, when trying to download from XNAT (ours is `version 1.8.3, build: 5` if that matters):

```
Traceback (most recent call last):
  File "/cm/shared/fsl-6.0.5/fslpython/envs/fslpython/lib/python3.8/site-packages/fsleyes/actions/browsexnat.py", line 220, in __onOk
    self.EndModal(wx.ID_OK)
wx._core.wxAssertionError: C++ assertion "Assert failure" failed at /home/conda/feedstock_root/build_artifacts/wxpython_1585592665613/work/ext/wxWidgets/src/gtk/dialog.cpp(199) in EndModal(): either wxDialog:EndModal called twice or ShowModal wasn't called

/cm/shared/fsl-6.0.5/bin/fsleyes: line 2: 34494 Segmentation fault      (core dumped) /cm/shared/fsl-6.0.5/fslpython/envs/fslpython/bin/python -I /cm/shared/fsl-6.0.5/fslpython/envs/fslpython/bin/fsleyes "$@"
```

**Comment by pauldmccarthy** [comment]
Hi @dmd, what version of wxPython are you using?

**Comment by dmd** [comment]
Hi Paul,

I *was* running `4.0.7.post2 gtk2 (phoenix) wxWidgets 3.0.5` but having remembered #77 I tried upgrading to `4.1.1 gtk2 (phoenix) wxWidgets 3.1.5`, and I still get the same error.

**Comment by pauldmccarthy** [comment]
Very strange - and this is installed from `conda-forge`? I can't reproduce this using either GTK2 or GTK3 (from pypi) builds of wxpython. 

Are you able to list the specific sequence of steps that lead to the error (or a screencast)? I.e. clicking on specific buttons, pressing enter, closing specific windows, etc..

**Comment by dmd** [comment]
Sent screen recording to your gmail.

**Comment by pauldmccarthy** [comment]
Resolved in pauldmccarthy/wxnatpy#2
